package.json

```diff
@@ -52,6 +52,7 @@
     "moment": "~2.10.6",
     "mongoose": "~4.3.4",
     "mongoose-validators": "~0.1.0",
+    "nodemailer": "^2.5.0",
```

Comment:
Please don't specify deps this way. If you don't want to pin them, use `~` instead of `ˆ`.

More info: http://fredkschott.com/post/2014/02/npm-no-longer-defaults-to-tildes/